capstone/test_data/test_fixtures/factories.py

```diff
@@ -48,13 +48,13 @@ def instance_fixture(db):
 
 
 @register
-class EmailBlocklistFactory(factory.DjangoModelFactory):
+class EmailBlocklistFactory(factory.django.DjangoModelFactory):
```

Comment:
Since `factory-boy` library was updated, it introduced breaking changes in the constructor when subclassing DjangoModelFactory.